nwbinspector/nwbinspector.py

```diff
@@ -200,24 +217,60 @@ def inspect_all_cli(
     n_jobs: int = 1,
     skip_validate: bool = False,
     detailed: bool = False,
+    stream: bool = False,
+    version_id: Optional[str] = None,
 ):
-    """Primary CLI usage of the NWBInspector."""
+    """
+    Run the NWBInspector via the command line.
+
+    path :
+    Relative path to either a local NWBFile, a local folder containing multiple NWBFiles, a link to a dataset on
+    DANDI archive (i.e., https://dandiarchive.org/dandiset/dandiset_id/version_id), or a six-digit DANDISet ID.
```

Comment:
while all dandiset ids are currently 6 digits, they are not constrained to always use that form

```suggestion
    DANDI archive (i.e., https://dandiarchive.org/dandiset/{dandiset_id}/{version_id}), or a Dandiset ID.
```